libs/dart/didcomm/atm_client/lib/src/messages/atlas/get_mediator_requests_message.dart

```diff
@@ -19,14 +18,13 @@ class GetMediatorRequestsMessage extends PlainTextMessage {
     super.body = const {},
     super.threadId,
   }) : super(
-          type: Uri.parse(
-            'affinidi.io/operations/ama/getMediatorRequests',
-          ),
+         operationName: 'getMediatorRequests',
```

Comment:
I propose we make it as a static final field on this class, please, to avoid magic strings.